cmd/lncli/cmd_pay.go

```diff
@@ -336,7 +343,13 @@ func sendPaymentRequest(ctx *cli.Context,
 	}
 
 	req.CltvLimit = int32(ctx.Int(cltvLimitFlag.Name))
-	req.TimeoutSeconds = paymentTimeoutSeconds
+
+	pmtTimeout := ctx.Duration("timeout")
```

Comment:
nit++: why not call this `paymentTimeout`?

Reply:
done 👍